src/modules/utils.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable-next-line */
+import dragListeners from './draggableLists.js';
+import updateStatus from './status.js';
+import Tasks from './Tasks.js';
+
+const clearCompletedBtn = document.querySelector('.clearBtn');
+
+const MyTasks = new Tasks();
+
+const selectTask = () => {
+  document.querySelectorAll('.task').forEach((e) => {
+    e.addEventListener('click', (e) => {
+      /* eslint-disable */
+
+      /* Get source of event */
+      const selectedTask = document.querySelector(
+        `[data-index="${e.originalTarget.dataset.index}"`
+      );
+
+      /* Remove highlight from all tasks */
+      document.querySelectorAll('.task').forEach((e) => {
+        e.classList.remove('selected');
+        e.removeChild(e.lastChild);
+        e.appendChild(
+          new DOMParser().parseFromString(
+            `<i class="fa-solid data-icon-index=${e.dataset.index} fa-ellipsis-vertical"></i>`,
+            'text/html'
+          ).body.childNodes[0]
+          /* eslint-enable */
+        );
+      });
+
+      /* Add highlight to selected task */
+      selectedTask.classList.add('selected');
+
+      /* show delete icon */
+      selectedTask.removeChild(selectedTask.lastChild);
+      /* eslint-disable */
```

Comment:
- [ ] It's advised to don't disable linters as it helps to write a good and clean code according to coding best practices. Kindly remove this line of code